src/lib/api/delegates/types.ts

```diff
@@ -97,6 +97,7 @@ export type DelegationEvent = {
   delegateeId?: string;
   delegatorId: string;
   eventDate: string;
+  eventTimestamp: number;
```

Comment:
I don't think this is a number from the API - appears to be a date string

<img width="348" height="58" alt="Screenshot 2025-08-08 at 2 03 18 PM" src="https://github.com/user-attachments/assets/471bb412-018d-4979-b497-a3984932dd17" />